openapi/src/parts/workspace.yaml

```diff
@@ -546,8 +546,6 @@ components:
         Request body for cloning an entire workspace. Cloning instructions
         are taken from individual resources.
       type: object
-      required:
-        - spendProfile
```

Comment:
Is it worth expanding the documentation of the `spendProfile` property below to state the fallback behavior when unspecified?

Reply:
Yes, I think it is. I'll add to it